### ISSUE TITLE
[5pt] Inundation Nation error

### ISSUE THREAD

**Issue by RobHanna-NOAA** [NOAA-OWP/inundation-mapping#1025]

**Comment by RyanSpies-NOAA** [comment]
Hey Rob - can you try using my `dev-fix-inundate-nation feature branch`?
https://github.com/NOAA-OWP/inundation-mapping/tree/dev-fix-inundate-nation

**Comment by RobHanna-NOAA** [comment]
Yup. I added the fix from this card to your feature branch and it seems good now.

**Comment by RobHanna-NOAA** [comment]
Testing a full BED (4.4.5.0)  on it now.
